template.md

```diff
@@ -0,0 +1,7 @@
+---
+  name: Your name
+  link: A link for your name. (Remove this line if you don't want a link.)
+  affiliation: Your affiliation. (Remove this line if you don't want an affiliation.)
+  title: My title. (Remove this line if you don't want to list your title.)
+  github: Your github username (This won't show on the page, but it helps us keep track of where signatures originated in case we have to commit your signature for you to avoid merge conflicts.)
```

Comment:
I can't imagine a case where there would be merge conflicts here. Two people making a file with the same name? But fine to leave in.